react/components/EditorContainer/Sidebar/ConfigurationList/List/Card/index.tsx

```diff
@@ -50,8 +64,17 @@ const Card = ({
           textColor="mid-gray"
         />
       </div>
+      <div
+        className="absolute top-0 right-0 mt1"
+        onClick={e => {
+          e.preventDefault()
+          e.stopPropagation()
```

Comment:
I would move this to an external function, in order to avoid its recreation on each render.